pkgs/development/python-modules/Kconfiglib/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub }:
+buildPythonPackage rec {
+  pname = "Kconfiglib";
```

Comment:
```suggestion
  pname = "kconfiglib";
```

Reply:
yet the correct name is `Kconfiglib` :p but well I changed it